be/src/models/query-list.model.ts

```diff
@@ -0,0 +1,10 @@
+import {DateTime} from '../generated';
```

Comment:
This is an internal model, so there should be no reference to the API-Layer.

Use `Date` and perform the conversion DateTime -> Date in the API-Layer